projects/components/src/quick-search/quick-search.component.spec.ts

```diff
@@ -637,317 +707,205 @@ describe('QuickSearchComponent', () => {
+                expect(this.quickSearch.getSelectedSearchResultItem().text()).toEqual('copy');
             });
 
             it('selects the last item when being at the top of the list and up is pressed', function (this: Test): void {
                 this.quickSearchData.simpleProvider.sectionName = 'new section';
                 this.quickSearchData.spotlightSearchService.registerProvider(this.quickSearchData.simpleProvider);
                 this.finder.hostComponent.spotlightOpen = true;
                 this.finder.detectChanges();
-                this.quickSearch.searchInputValue = 'c';
+                this.quickSearch.getInput().type('c');
                 // Test
-                expect(this.quickSearch.getSelectedItem(1)).toEqual('copy');
+                expect(this.quickSearch.getSelectedSearchResultItem().text()).toEqual('copy');
                 // Press arrow up key
-                this.quickSearch.pressArrowUp();
+                this.quickSearch.getInput().sendKeyboardEvent('keydown', { key: 'ArrowUp' });
                 // Test
-                expect(this.quickSearch.getSelectedItem(2)).toEqual('create');
+                expect(this.quickSearch.getSelectedSearchResultItem().text()).toEqual('create');
             });
         });
-    });
 
-    describe('item handler', () => {
-        it('is called when pressing enter and there is a selection', function (this: Test): void {
-            const itemHandlerSpy = spyOn(this.quickSearchData.simpleProvider, 'itemHandler').and.callThrough();
-            this.finder.hostComponent.spotlightOpen = true;
-            this.finder.detectChanges();
-            this.quickSearch.searchInputValue = 'c';
-            this.quickSearch.pressEnter();
-            expect(itemHandlerSpy).toHaveBeenCalledWith('copy');
-        });
-
-        it('is not called when pressing enter and there is no selection', function (this: Test): void {
-            const itemHandlerSpy = spyOn(this.quickSearchData.simpleProvider, 'itemHandler').and.callThrough();
+        describe('item handler', () => {
+            it('is called when pressing enter and there is a selection', function (this: Test): void {
+                const itemHandlerSpy = spyOn(this.quickSearchData.simpleProvider, 'itemHandler').and.callThrough();
+                this.finder.hostComponent.spotlightOpen = true;
+                this.finder.detectChanges();
+                this.quickSearch.getInput().type('c');
+                this.quickSearch.getInput().sendKeyboardEvent('keydown', { key: 'enter' });
+                expect(itemHandlerSpy).toHaveBeenCalledWith('copy');
+            });
 
-            // Prepend async provider so that there is no auto selection
-            Object.assign(this.quickSearchData.asyncProvider, { order: 0 });
+            it('is not called when pressing enter and there is no selection', function (this: Test): void {
```

Comment:
Something weird going on with the diff here... let's see if we can make this go away